doc/CHEATSHEET.md

```diff
@@ -2767,3 +2767,30 @@ try (KnativeClient kn = new DefaultKnativeClient()) {
     kn.services().inNamespace("default").createOrReplace(service);
 }
 ```
+
+#### Logging
+Using logging-interceptor:
+
+- Configure OkHTTP logging:
+``` 
+- Set logging level to trace in my simplelogger.properties file:
+```
+ org.slf4j.simpleLogger.defaultLogLevel=trace
+
+- Run following code and get the messsages sent and recived from/to the  kubernetes client:
+```
+try (OpenShiftClient client = new DefaultOpenShiftClient()) {
+    SelfSubjectAccessReview ssar = new SelfSubjectAccessReviewBuilder()
+            .withNewSpec()
+            .withNewResourceAttributes()
+            .withGroup("apps")
+            .withResource("deployments")
+            .withVerb("create")
+            .withNamespace("rokumar")
+            .endResourceAttributes()
+            .endSpec()
+            .build();
+
+    ssar = client.authorization().v1().selfSubjectAccessReview().create(ssar);
```

Comment:
I think we can remove this SelfSubjectAccessReview example. You can just show the verbose logs output or maybe just keep it like this:

Using logging-interceptor:

- Set logging level to trace in my simplelogger.properties file would print okhttp requests and response bodies from apiServer:
```
 org.slf4j.simpleLogger.defaultLogLevel=trace
```

Reply:
![Untitled](https://user-images.githubusercontent.com/79860504/119133220-45276f00-ba59-11eb-92bc-87d98f3e0aa4.png)
Will this work?

Reply:
Also how i can be added to mailing list?
Or a channel where i can ask questions?

Reply:
I think this should be good enough. However, you have code blocks around line 2776, they should instead be around 2778